examples/unattended_baggage/demo.ipynb

```diff
@@ -0,0 +1,357 @@
+    "!wget https://github.com/Megvii-BaseDetection/YOLOX/releases/download/0.1.1rc0/yolox_x.pth\n",
+    "!cd YOLOX && pip3 install .\n",
+    "# download VQPy, move vqpy/ to root directory for import\n",
+    "!git clone https://github.com/uclasystem/VQPy.git\n",
+    "!mv VQPy/vqpy ./\n",
+    "# install VQPy's dependencies\n",
+    "!pip3 install lap cython_bbox shapely\n",
+    "import vqpy\n",
+    "video_path = \"./baggage.mp4\"\t# or change to wherever you put the video\n",
+    "save_folder = \"./vqpy_outputs\"\n",
+    "detector_model_dir = \"./\""
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For reference, the working directory should look like:\n",
+    "\n",
+    "```text\n",
+    ".\n",
+    "├── VQPy\n",
+    "├── YOLOX\n",
+    "├── baggage.mp4\t# video to query on\n",
+    "├── vqpy\t# make vqpy available for import\n",
+    "└── yolox_x.pth\t# model checkpoint\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Unattended baggage query with VQPy"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 1: Define `VObj` type for person\n",
+    "\n",
+    "First, define a `Person` `VObj`. Since we only need two built-in properties:\n",
+    "\n",
+    "- bounding box (`tlbr`), for computing distance between person and baggage\n",
+    "- id (`track_id`), for referencing `Person` from baggage `VObj`\n",
+    "\n",
+    "`Person` can just inherit `VObjBase`, with no extra property functions defined."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Person(vqpy.VObjBase):\n",
+    "    pass"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define VObj type for baggage with property `owner`\n",
+    "\n",
+    "Next define a `Baggage` `VObj`. To query baggages without owners, we add a property `owner` to `Baggage`.\n",
+    "\n",
+    "In this example, the baggage's `owner` references `track_id` of `Person` VObj, who is:\n",
+    "\n",
+    "- the owner from last frame, if person is still present and within `DISTANCE_THRESHOLD`\n",
+    "- else, person closest to the baggage and within `DISTANCE_THRESHOLD`\n",
+    "- else, None\n",
+    "\n",
+    "With the above logic:\n",
+    "\n",
+    "- `owner` should be decorated with `@stateful` so that baggage's owner from last frame can be accessed\n",
+    "\n",
+    "\t`@stateful()` will store all past values of property being decorated to the VObj, allowing us to use `getv(\"owner\", -2)` to access value of `owner` of last frame.\n",
+    "\n",
+    "- Decorator `@cross_vobj_property` should be applied to `owner`, since `owner` needs information from other VObjs (`Person`s) in the current frame.\n",
+    "\n",
```

Comment:
The arguments for `@cross_vobj_property` are
* `vobj_type`:
* `vobj_input_fields`.  explain about id, tlbr
The `person_ids_tlbrs` value will be properties of all the `Person` VObjs on the frame. ([`person_1_properties`, `person_2_properties`, ..., `person_n_peroperties`]), where `person_i_properties` is a tuple of the input field values: (id, tlbr) 


(You can rephrase as you want)